lib/RuntimeGlobals.js

```diff
@@ -405,3 +405,23 @@ module.exports.uncaughtErrorHandler = "__webpack_require__.oe";
  * an object containing all installed WebAssembly.Instance export objects keyed by module id
  */
 module.exports.wasmInstances = "__webpack_require__.w";
+
+/**
+ * the webpack id of the chunk
+ */
+module.exports.webpackId = "__webpack_id__";
+
+/**
+ * the webpack ids of the chunk
+ */
+module.exports.webpackIds = "__webpack_ids__";
+
+/**
+ * the webpack modules of the chunk
+ */
+module.exports.webpackModules = "__webpack_modules__";
+
+/**
+ * the webpack runtime of the chunk
+ */
+module.exports.webpackRuntime = "__webpack_runtime__";
```

Comment:
Do we use these variables somewhere else? I think `__webpack_modules__` can be replaced in more place, can you check it?